hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -127,6 +136,9 @@ public void doStateTransition() {
             long balance = op.getInitialBalance();
             final var customizer = asCustomizer(op);
             final var created = ledger.create(sponsor, balance, customizer);
+            if (op.getAlias().size() == EVM_ADDRESS_SIZE) {
```

Comment:
Since [we allow](https://github.com/hashgraph/hedera-services/blob/04108-pre-charge-hollow-account-finalization/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java#L392) the possibility of someone setting both an EVM address _and_ the matching ECDSA key, don't we want,
```
    if (op.getAlias().size() == EVM_ADDRESS_SIZE && !op.hasKey()) {
       ...
```

Reply:
🤦🏽‍♂️  Of course we do, changed now.